components/pkg-export-docker/defaults/Dockerfile.hbs

```diff
@@ -1,6 +1,15 @@
 FROM {{base_image}}
 ENV PATH {{path}}
-ADD {{rootfs}} /
+
+ADD {{rootfs}}/root /root
+ADD {{rootfs}}/tmp /tmp
+ADD {{rootfs}}/var /var
+ADD {{rootfs}}/etc /etc
+{{~ #each packages as |pkg|}}
+ADD {{../rootfs}}/hab/pkgs/{{pkg}} /hab/pkgs/{{pkg}}
```

Comment:
I see each package is going to its own layer, but I'm not sure is it good idea from performance point of view to have such many layers? 
From docker documentation:
> The overlay2 driver natively supports up to 128 lower OverlayFS layers

Is it possible to package have such number of TDEPS? Probably yes

Issue: https://github.com/moby/moby/issues/26380

Reply:
UPD2: After googling and thinking looks like nothing can be done here, probable just show a warning when there are more than 100 packages in TDEPS

Reply:
@jsirex Agreed. I'll have an updated version up soon that will let you choose whether to use a package-per-layer, or use the existing everything-in-one-layer behavior, which remains the default. That way, if you want reuse, you can have it, but if that results in too many layers, there's still a way to build a container.